### PR TITLE
NumPad component. Fix hover color on buttons. Refactor SASS variables to remove 'hemohub' and use bootstrap naming convention.

### PR DESCRIPTION
**Pull request by olgapuig**
Refactor SASS variables to remove 'hemohub' and use bootstrap naming convention.
- Rename $primary-color to $primary
- Rename $background-hemohub to $background-secondary
- Rename $background to $background-primary

### CODE REVIEW

**Comment by olgapuig** [comment]
Fix hover color for all buttons (maximized and fixed size dialogs)

**Comment by olgapuig** [comment]
New NumPad component

File: src/app/systelab-components/numpad/numpad.component.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	selector: 'systelab-numpad',
+	templateUrl: 'numpad.component.html'
+})
+export class NumPadComponent  {
+	@Input()
+	public numpadValue = '';
+	@Input()
+	isPassword = false;
+
+	/*set numpadValue(numpadValue: string) {
```
Comment (carlosra85) [inline src/app/systelab-components/numpad/numpad.component.ts]: Remove unnecessary commented methods

File: src/app/systelab-components/numpad/numpad.component.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		const parameters: NumPadDialogParameters = NumPadDialog.getParameters();
+		parameters.containerLabel = this.numpadValue;
+		parameters.isPassword =  this.isPassword;
+		this.dialogService.showDialog(NumPadDialog, parameters)
+			.subscribe( response => {
+					this.numpadValue = response;
+				});
+	}
+
+	updateContainerLabel(containerLabel: string) {
```
Comment (carlosra85) [inline src/app/systelab-components/numpad/numpad.component.ts]: Unused method

File: src/app/systelab-components/numpad/numpad.dialog.component.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            <div class="col-md-4 nopadding">
+                <a href="#" class="btn form-control" data-key="48" (click)="pushButton($event)">0</a>
+            </div>
+            <div class="col-md-4 nopadding">
+                <a href="#" class="btn form-control" data-key="51" (click)="deleteNumber()"><</a>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-md-12 nopadding">
+                <a href="#" class="btn form-control" data-key="8" (click)="accept()">Aceptar</a>
```
Comment (carlosra85) [inline src/app/systelab-components/numpad/numpad.dialog.component.html]: Aceptar should be a translation variable

File: src/app/systelab-components/numpad/numpad.dialog.component.scss [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+.container {
+  margin-right: auto;
+  margin-left: auto;
+}
+.inputNumpad{
+  height: 60px;
+  font-size: 42px;
+  line-height: 35px;
+  width: 100%;
+}
```
Comment (carlosra85) [inline src/app/systelab-components/numpad/numpad.dialog.component.scss]: Custom styles must start with slab- to differentiate from bootstrap styles. Do not overwrite bootstrap styles. Try yo use variables and dynamic heights when possible

File: src/app/systelab-components/numpad/numpad.dialog.component.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+	public deleteNumber() {
+		this.containerDialogLabel = this.containerDialogLabel.substring(0, this.containerDialogLabel.length - 1);
+	}
+
+	public cleanInput() {
+		this.containerDialogLabel = '';
+	}
+}
+
+
```
Comment (carlosra85) [inline src/app/systelab-components/numpad/numpad.dialog.component.ts]: Use a more generic name for the input text variables than containerLabel, containerDialogLabel and containerDialogLabelOld. Initially the numPad was designed for container labels but it is used for any number field

**Pull request merged**